CHANGELOG.md

```diff
@@ -0,0 +1,41 @@
+# Unreleased features
+
+(None yet)
+
+# 0.3.0 (2024-08-13)
+
+New features:
+
+- Integrated libp2p for transport encryption and authentication of peers.
+- Added a password protection – peers must specify the same `--secret` to connect.
+- You can now provide default startup arguments in `.ethersync/config`.
+- When editors send
```

Comment:
when editors send! :D love letters? 💌 

do you remember what you wanted to write here? could it be in a similar direction to what I added in 0.2.2 regarding editor `open`ing? we have certainly improved this between 0.2.2 and 0.3.0

Reply:
I don't remember :cry: Yes, I like adding that here :)